algorithms/merging/merge.py

```diff
@@ -279,3 +283,134 @@ def truncate(merged_intensities):
     logger.info("Total number of rejected intensities %s", n_removed)
     logger.debug(out.getvalue())
     return amplitudes, anom_amplitudes
+
+
+def dano_over_sigdano_stats(anomalous_amplitudes, n_bins=20):
+    """Calculate the statistic for resolution bins and overall."""
+    vals = flex.double()
+    # First calculate dF/s(dF) per resolution bin
+    anomalous_amplitudes.setup_binner(n_bins=n_bins)
+    resolution_bin_edges = flex.double()
+    for i_bin in anomalous_amplitudes.binner().range_used():
+        sel = anomalous_amplitudes.binner().selection(i_bin)
+        arr = anomalous_amplitudes.select(sel)
+        vals.append(dano_over_sigdano(arr))
+        resolution_bin_edges.append(anomalous_amplitudes.binner().bin_d_min(i_bin))
+    resolution_bin_edges.append(anomalous_amplitudes.binner().d_min())
+    return vals, resolution_bin_edges
+
+
+def dano_over_sigdano(anomalous_amplitudes):
+    """Calculate < |F(+) - F(-)|> / <sigma(F(+) - F(-))> i.e. DANO/SIGDANO."""
+    diff = anomalous_amplitudes.anomalous_differences()
+    if not diff.data() or not diff.sigmas():
+        return 0.0
+    return flex.mean(flex.abs(diff.data())) / flex.mean(diff.sigmas())
+
+
+def print_dano_table(anomalous_amplitudes):
+    """Calculate dano/sigdano in resolution bins and print to the logger."""
+    dFsdF, resolution_bin_edges = dano_over_sigdano_stats(anomalous_amplitudes)
+
+    logger.info("Size of anomalous differences")
+    header = ["d_max", "d_min", "dano/sigdano"]
+    rows = []
+    for i, dF in enumerate(dFsdF):
+        rows.append(
+            [
+                f"{resolution_bin_edges[i]:6.2f}",
+                f"{resolution_bin_edges[i+1]:6.2f}",
+                f"{dF:6.3f}",
+            ]
+        )
+    logger.info(tabulate(rows, header))
```

Comment:
It might be better to return either the table, or string representation here and move the `logger.info` calls to the command line script, which would potentially make this function more useful elsewhere.